### DISCUSSION TITLE
Parsing error: Unexpected token

### DISCUSSION THREAD

**Discussion by gregarega1993** [tailwindlabs/tailwindcss#5626] [Help]
When writing this I get an error: "Parsing error: Unexpected token" where function ({ addComponents }) { is written. What is the issue?

`plugins: [
    require('@tailwindcss/forms'), 
    require('@tailwindcss/typography'),
    function ({ addComponents }) {
      addComponents({
        '.container': {
          width: '100%',
          '@screen sm': {
            maxWidth: '640px',
          },
          '@screen md': {
            maxWidth: '768px',
          },
          '@screen lg': {
            maxWidth: '1024px',
          },
          '@screen xl': {
            maxWidth: '1280px',
          },
        }
      })
    }
  ],`

**Comment by simonswiss** [comment]
Hey!

I think you need to wrap your function in a `plugin()` function. I just changed the `.container` class to `.test`, since a `container` class already exists in Tailwind:

```js
// tailwind.config.js

plugins: [
  require('@tailwindcss/forms'),
  require('@tailwindcss/typography'),
  plugin(function ({ addComponents }) {
    addComponents({
      '.test': {
        width: '100%',
        '@screen sm': { maxWidth: '640px' },
        '@screen md': { maxWidth: '768px' },
        '@screen lg': { maxWidth: '1024px' },
        '@screen xl': { maxWidth: '1280px' },
      },
    })
  }),
]
```

Here's a Tailwind Play example with your code wrapped in the `plugin()` function:

https://play.tailwindcss.com/8NltvUTfw0?file=config

If that was not your issue, you should probably share some code in a repo so we can have a look.

Hope it helps!
